### PR TITLE
More informations in webhook

### PR DESCRIPTION
**Pull request by nathanaelhoun**
## Changes

- [x] This goes with the [release of the orb](https://circleci.com/developer/orbs/orb/nathanaelhoun/mattermost-plugin-notify)
- [ ] Update to README.md and help messages in plugin will go in another PR

### CODE REVIEW

**Comment by nathanaelhoun** [comment]
image

File: server/plugin/webhook.go [hunk 1]
```diff
@@ -38,8 +39,9 @@ func (wi *WebhookInfo) ToPost(buildFailedIconURL, buildGreenIconURL string) *mod
 				Title: "Repo",
 				Short: true,
 				Value: fmt.Sprintf(
-					"[%s](%s)",
-					v1.GetFullNameFromOwnerAndRepo(wi.Owner, wi.Repository),
+					"[%s/%s](%s)",
+					wi.Organization,
```
Comment (darkLord19) [inline server/plugin/webhook.go:43]: Do we want to replace GetFullNameFromOwnerAndRepo elsewhere in code with Organization?

**Comment by darkLord19** [comment]
Great work @nathanaelhoun  :)

**Review by darkLord19 — APPROVED** [review]

File: server/plugin/webhook.go (hunk 1 above)
Comment (nathanaelhoun) [reply server/plugin/webhook.go:43]: I think this method will disappear. It was used to store the information, and here, it is was used for displaying. So I separated it.

**Pull request merged**